articles/is-innersource-right.md

```diff
@@ -0,0 +1,33 @@
+# Is InnerSource Right for My Project?
+
+InnerSource is the application of open source principles to company-internal software development. Done right, it unblocks progress and eases adoption of shared services and modules. This article contains guidance and questions to ask yourself when considering adoption of an InnerSource approach to running your project.
+
+## Will Contributions Come?
+
+An InnerSource approach only makes sense if contributions are expected from the project's users. You can expect contributions to come if you see or anticipate noticeable amounts of energy directed at your project space by its users. Some examples:
```

Comment:
What does "project space" mean? Why not just "project"?

Reply:
If users are home-rolling workarounds that are in your project space (but not necessarily your exact project) then that is also a good reason to run your project as InnerSource.